scripts/vsc-venv-wrapper/README.md

```diff
@@ -0,0 +1,82 @@
+# Python Virtual Environment Wrapper Script
+
+`vsc-venv` is a script that encapsulates the creation and management of Python virtual environments. 
+This avoids multiple issues with the default `venv` command on the HPC. 
+
+
+One issue is that a virtual environment created for one cluster might not work on another. 
+Additionally, when activating a virtual environment, 
+the same centrally installed modules used during its creation must also be loaded.
+the `vsc-venv` command manages multiple virtual environments for different clusters in a transparent way, 
+while guaranteeing the same module environment.
+
+## Usage
+
+A virtual environment can be activated by running the following command:
+
+```bash
+source ./vsc-venv.sh activate <requirements> [module_script]
+```
+
+Here, `requirements` is the path to a file containing the python dependencies to install in the virtual environment.
+An optional module script can be provided to load the required modules before creating the virtual environment.
+
+Automatically, the modules are loaded and the environment is activated. 
+When running this command for the first time, the dependencies from the requirement file are installed.
+
+Now, the software can be run and use packages installed in the virtual environment, along with centrally installed modules.
+To deactivate the virtual environment, run:
+
+```bash
+source ./vsc-venv.sh deactivate
+```
+
+## Example
+
+Suppose we are on the login node, and we want to make a virtual environment for doduo, which is loaded by default.
+The following files are present in the current directory:
+
+**modules.sh:**
+```bash
+ml Python/3.12.3-GCCcore-13.3.0
+```
+
+and **requirements.txt:**
```

Comment:
add a ref to the supported syntax (eg version locks/limits)

Reply:
Added in [requirements.txt docs referene added](https://github.com/hpcugent/vsc_user_docs/pull/670/commits/210cb01434b465a6ec2b261294d44d896ea4e4b7)